src/pieces_os_client/wrapper/copilot.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, pieces_client: "PiecesClient"):
         self.context = Context(pieces_client)
         self._chat = None
         self._chat_id = None
+        self.context_response:Optional[RelevantQGPTSeeds] = None
```

Comment:
Is `context_response` a good name for this? What is this field for?

Reply:
Storing the response of the relevance endpoint

Reply:
@jimbobbennett should I change the name?

Reply:
Can you suggest another name then?

Reply:
How about `relevant_context`?